cypress/integration/run-processes/run-import-d2cc-process-acceptance-test.js

```diff
@@ -30,31 +31,29 @@ Cypress.on('uncaught:exception', (err, runnable) => {
 
 describe('Run d2cc process', () => {
 
-   it('Run a full d2cc process ', () => {
+    it('Run a full d2cc process ', () => {
```

Comment:
It will be nice to mention that this is the import process

Reply:
renamed